### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by bradymiller**
some misc updates/fixes:
php8.0
php8.1
uuid
upgrade

### CODE REVIEW

**Comment by stephenwaite** [comment]
cool-cat

File: interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php [hunk 1]
```diff
@@ -550,7 +550,7 @@ public function getMedications($pid, $encounter)
 
     public function getProblemList($pid, $encounter)
     {
-        (new UuidRegistry(['table_name' => 'lists']))->createMissingUuids();
+        UuidRegistry::createMissingUuidsForTables(['lists']);
```
Comment (sjpadgett) [inline interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php:553]: I couldn't get this to work! strange... I'm going to add back in my version too be sure.

Comment (adunsulag) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php:553]: What error were you hitting?

Comment (sjpadgett) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php:553]: This works and nope unsure why. Maybe updated environment.

**Comment by bradymiller** [comment]
testing well and bringing in

**Pull request merged**